readFile.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import csvparser from 'csv-parser';
+export async function readFile(fileName, day, hour) {
+    let result = {
+        list: [], day: day, hour: hour
+    };
+    return new Promise((resolve, reject) => {
+        fs.createReadStream(fileName)
+            .pipe(csvparser())
+            .on('data', function (data) {
+                try {
+
+                    if (data.date == day) {
+
+                        result.list.push(data);
+                    }
+
+                }
+
+                catch (err) {
+
+                    console.log('Error: ' + err);
+                    reject(err);
+                }
```

Comment:
obsolete line breaks make the code here a bit harder to read, but that's just a visual suggestion